st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -48,6 +52,17 @@ class ActionAliasController(resource.ContentPackResourceController):
         'sort': ['pack', 'name']
     }
 
+    _custom_actions = {
```

Comment:
Actually, since this is a post we should probably send data as body and not via query string.

This way it will be consistent with other models. Yes, it's a bit "heavy" and we need a wrapper API model, but we already do that in other places.

I will make that change, hope you don't mind :)